.circleci/config.yml

```diff
@@ -1,30 +1,64 @@
 # Golang CircleCI 2.0 configuration file
 #
 # Check https://circleci.com/docs/2.0/language-go/ for more details
-version: 2
+version: 2.1
 jobs:
-  build:
+  generate_swagger:
+    environment:
+      SWAGGER_VERSION=2.4.9
```

Comment:
I think it's better to use swagger version from file, it's automatically updated when rendering api client locally.

Reply:
True, but I don't have this information here because I don't have our repository code here.

Reply:
Fixed.